docs/Deployment/ManualDeployment.md

```diff
@@ -125,28 +125,30 @@ You need to create an account on Cloudflare and add their DNS to [your domain](h
 
 ### Step 2 Setting up Argo Tunnel
 
-After you've added the DNS and is propagated, you can start the setup of the Argo tunnel. Go to [Zero Trust](https://dash.teams.cloudflare.com/c1cd2ab2129af8897457d6f97c082725/onboarding) option on the left menu, go to "access", then click "tunnels".
+After you've added the DNS and is propagated, you can start the setup of the Argo tunnel. 
+Go to [Zero Trust](https://dash.teams.cloudflare.com/c1cd2ab2129af8897457d6f97c082725/onboarding) option on the left menu, go to "access", then click "tunnels".
```

Comment:
Guys there is a link to an account that is wrong, the string after https://dash.teams.cloudflare.com/ is identifying an account, please remove that.

Reply:
Took this out in last update :) TY!